app/lib/frontend/templates.dart

```diff
@@ -104,6 +104,8 @@ class TemplateService {
 
     final PlatformDict platformDict = getPlatformDict(currentPlatform);
     final isSearch = searchQuery != null && searchQuery.hasQuery;
+    final unsupportedQualifier =
+        isSearch && searchQuery.parsedQuery.text.contains(':');
```

Comment:
I'm guessing we should support a literal `"cool:beans"` we want to support that, but it'd be caught here?

...part of making this sarter?

Reply:
The `parsedQuery.text` is the remainder, it is whatever is left over processing the other known `X:Y` patterns. Right now `"explicit phrases"` are not part of that processing (although they should), but it could be addressed in a different/unrelated PR.

Reply:
kk